Makefile.am

```diff
@@ -320,8 +322,7 @@ update-po: missing-languages
 
 missing-languages:
 	bash -c '\
-	eval $$(grep ALL_LINGUAS configure.ac); \
-	diff -u <(echo $$ALL_LINGUAS | xargs -rn1 echo) \
+	diff -u <(cat po/LINGUAS) \
```

Comment:
@inigomartinez I guess this is to fix occasional build error when two langs switched places in the output, correct?